cmd/s3-gw/app.go

```diff
@@ -207,3 +210,45 @@ func (a *App) Server(ctx context.Context) {
 
 	close(a.webDone)
 }
+
+func getCacheOptions(v *viper.Viper, l *zap.Logger) *layer.CacheConfig {
+	cacheCfg := layer.CacheConfig{
+		ListObjectsLifetime: layer.DefaultObjectsListCacheLifetime,
```

Comment:
Maybe move default cache config values to the app-settings.go file?

Reply:
I think it would be better to store consts for cache in files with cache implementation...